### PR TITLE
Set net type on constant nets.

### PR DESCRIPTION
**Pull request by litghost**

### CODE REVIEW

**Review by mithro — APPROVED** [review]
LGTM.

File: fasm2bels/lib/interchange_capnp.py [hunk 1]
```diff
@@ -522,6 +531,8 @@ def finish_encode(self):
             for stub_obj, stub in zip(net.stubs, stubs):
                 stub.output_interchange(stub_obj, self.string_id)
 
+            net.type = net_to_type.get(net_name, 'signal')
```
Comment (acomodi) [inline fasm2bels/lib/interchange_capnp.py:534]: Where is net.type used, and why should it default to signal?

**Review by acomodi — APPROVED** [review]
LGTM

File: fasm2bels/lib/interchange_capnp.py (hunk 1 above)
Comment (litghost) [reply fasm2bels/lib/interchange_capnp.py:534]: It's part of the FPGA interchange output.  I'm working on moving the FPGA interchange stuff to its own library, and it has better docs.

**Pull request merged**